Content.Client/Fax/UI/FaxBoundUi.cs

```diff
@@ -52,8 +52,27 @@ private async void OnFileButtonPressed()
         }
 
         using var reader = new StreamReader(file);
+
+        var label = await reader.ReadLineAsync();
         var content = await reader.ReadToEndAsync();
-        SendMessage(new FaxFileMessage(content[..Math.Min(content.Length, FaxFileMessageValidation.MaxContentSize)], _window.OfficePaper));
+
+        if (label is { })
+        {
+            if (label.StartsWith('#'))
+            {
+                label = label[1..].Trim();
+            }
+            else
+            {
+                content = label + '\n' + content;
+                label = null;
+            }
+        }
+
+        SendMessage(new FaxFileMessage(
+            label?[..Math.Min(label.Length, FaxFileMessageValidation.MaxLabelSize)],
+            content[..Math.Min(content.Length, FaxFileMessageValidation.MaxContentSize)],
+            _window.OfficePaper));
     }
```

Comment:
Isn't this always going to eat the first line?

Reply:
It will consume the first line separately, but if the line does not start with "#" it gets prepended to `content`. I had it using `StreamReader.Peek()` previously but I could not find documentation on whether that could cause delays since it is not asynchronous; and the rest of the line is going to be read anyway. The change is [6fc8763](https://github.com/space-wizards/space-station-14/pull/25979/commits/6fc8763c801e4e8613af9dd8370af35ca8698f77).

Reply:
I can rename `label` to something like `firstLine`; then it is conditionally parsed into `label` or prepended to `content`.